src/components/Question.jsx

```diff
@@ -119,25 +119,46 @@ export default class Question extends React.Component {
     this.setState({ showIcon: false });
   };
 
-  render() {
+  _getHeaderClassName = () => {
     const question = this.props.question;
-    const subQuestion = question[Constants.HAS_SUBQUESTION];
+    const collapsible = this.props.collapsible;
     const options = this.context.options;
-    const questionComponent = this.renderQuestion(question);
-    if (FormUtils.isHidden(question)) {
-      return null;
-    }
+    const subQuestion = question[Constants.HAS_SUBQUESTION];
```

Comment:
I understand that this is an array of subQuestions, thus I suggest subQuestion --> subQuestions?

Reply:
@LaChope u did not answer here

Reply:
Done